common/build.rs

```diff
@@ -1,8 +1,8 @@
 fn main() -> Result<(), Box<dyn std::error::Error>> {
-    println!("cargo:rerun-if-changed=proto");
+    println!("cargo:rerun-if-changed=../proto");
```

Comment:
Perhaps we can move it to `backend` now? The idea was to re-use generated stuff between frontend and backend, but now only `proto` files themselves are shared, generated code is not.

Reply:
It would likely be more efficient to postpone this task to the next pull request as it would introduce too many unrelated changes